### ISSUE TITLE
Remove support for Qt docks

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#1672]
Work is being done in the no-docks branch.

Don't panic. This branch will be merged only after careful consideration. Furthermore `git checkout 96f2826` will restore the legacy code.

The effect of this change will be to obsolete the pyzo_in_leo and pyzo_file_browser plugins. Imo, neither plugin is worth "saving".

**Comment by edreamleo** [comment]
The no-docks branch has been merged into devel.
